### PR TITLE
More ssh agent information

### PR DESCRIPTION
**Pull request by dannysauer**
More information, but not necessarily better information. :)

It's a start based on discussion in #245 , and there's lots of room for feedback.  That "tip" section is getting kind of out of hand, in particular, and I'm not positive if the PR should even be against `master`.  Nor am I quite sure how to format much of anything. :D

### CODE REVIEW

**Comment by nkoranova** [comment]
I would say this is now fool proof and if @bergmannf has no objections to the content, I would go ahead and merge it. :) Thanks for your contributions everyone!

**Comment by dannysauer** [comment]
There are a couple of spelling errors. One sec. :)

File: adoc/deployment-preparation.adoc [hunk 1]
```diff
@@ -4,37 +4,59 @@
...
+. The `ssh-agent` is usually started automatically by graphical
+desktop environments. If that is not your case run:
++
+----
+eval "$(ssh-agent)"
+----
+This will start the agent and set environment variables used for agent
+communication within the current session. This has to be the same terminal session
+that you run the `skuba` commands in. A new terminal usually requires a new ssh-agent.
+. In some desktop environments the ssh-agent will also automatically load the SSH keys.
```
Comment (dannysauer) [inline adoc/deployment-preparation.adoc]: Remove extra period / space
```suggestion
In some desktop environments the ssh-agent will also automatically load the SSH keys.
```

File: adoc/deployment-preparation.adoc [hunk 2]
```diff
@@ -4,37 +4,59 @@
...
+To add an SSH keys manually, use the `ssh-add` command:
```
Comment (dannysauer) [inline adoc/deployment-preparation.adoc]: singular "key"
```suggestion
To add an SSH key manually, use the `ssh-add` command:
```

File: adoc/deployment-preparation.adoc [hunk 3]
```diff
@@ -4,37 +4,59 @@
...
-You can load as many keys as you want into your agent using the
-`ssh-add <path to key>` command. Keys can also be password protected.
+If you are adding the SHH key manually, specify the full path.
+For example: `/home/sles/.ssh/id_rsa`
+====
 
-You can list all the identities loaded into the agent by using the
-`ssh-add -L` command.
+You can load multiple keys into your agent using the `ssh-add <path_to_key>` command.
+Keys should be password protected for security measures. The
```
Comment (dannysauer) [inline adoc/deployment-preparation.adoc]:
```suggestion
Keys should be password protected as a security measure. The
```

File: adoc/deployment-preparation.adoc [hunk 4]
```diff
@@ -4,37 +4,59 @@
...
-host to another one. This can be useful if you intend to run skuba on
+==== Forwarding the authentication agent connection
+It is also possible to *forward the authentication agent connection* from a
+host to another one, which can be useful if you intend to run skuba on
 a "jump host" and don't want to copy your private key to this node.
-
 This can be achieved using the `ssh -A` command. Please refer to the man page
-of ssh to learn about the security implications of using this feature.
-====
+of SSH to learn about the security implications of using this feature.
```
Comment (dannysauer) [inline adoc/deployment-preparation.adoc]: This is referring to the command `ssh`, not the protocol "SSH", so lower-case might be more accurate.  Maybe?  I'm somewhat indifferent.

File: adoc/deployment-preparation.adoc [hunk 5]
```diff
@@ -4,37 +4,59 @@
...
-The `ssh-agent` process is usually started automatically by most UNIX
-environments. If that's not the case, invoke the `ssh-agent` command
-and follow the guidance given by the tool's output.
-
-You can start the `ssh-agent` in the background by using the
-`eval "$(ssh-agent -s)"` command.
-
-You can load as many keys as you want into your agent using the
-`ssh-add <path to key>` command. Keys can also be password protected.
+If you are adding the SHH key manually, specify the full path.
```
Comment (dannysauer) [inline adoc/deployment-preparation.adoc]:
```suggestion
If you are adding the SSH key manually, specify the full path.
```

**Comment by dannysauer** [comment]
@nkoranova - I made a few minor tweaks (see closed review comments), and am not sure about an SSH/ssh capitalization thing.  Otherwise, I'm happy with your rewrite.  Feels weird to approve a PR I opened, though, so I'll just leave this "approve" comment. :D 

Also, it's interesting that code changes committed inline from the review process didn't undo the existing approval.

File: adoc/deployment-preparation.adoc (hunk 4 above)
Comment (stefsuse) [reply adoc/deployment-preparation.adoc]: usually man pages are referred to in lowercase, or even more specifically `ssh(1)`.

**Comment by nkoranova** [comment]
Great! Thanks @dannysauer for the review. And yes, @stefsuse was right about the ssh formatting, so I went and corrected it. https://doc.opensuse.org/documentation/styleguide/#sec.other_external 
I'll go and merge this now. :)

**Pull request merged**